components/grade-form.js

```diff
@@ -0,0 +1,19 @@
+class GradeForm {
+  constructor(formElement) {
+    this.formElement = formElement;
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.formElement.addEventListener('submit', this.handleSubmit);
+  }
+  onSubmit(createGrade) {
+    this.createGrade = createGrade;
```

Comment:
If you are going to add a property to your class, you need the property to be present in your constructor from instantiation. You can start it with a value of `null`

Reply:
Changes made, thanks for the feedback!